rules/azurerm_mysql_server_invalid_version.go

```diff
@@ -26,7 +24,7 @@ func NewAzurermMysqlServerInvalidVersionRule() *AzurermMysqlServerInvalidVersion
 		attributeName: "version",
 		enum: []string{
 			"5.7",
-			"8.0.21",
+			"8.0",
 		},
```

Comment:
think we should consider dropping rules when automation breaks rather than switching them to be manually maintained since this enum will become outdated